pkg/meta/annotations.go

```diff
@@ -48,9 +48,17 @@ const (
 	// This is a feature flag for mounting NMA certs as a secret volume in server containers
 	// if deployment method is vclusterops. When set to true the NMA reads certs from this mounted
 	// volume, when set to false it reads certs directly from k8s secret store.
-	MountNMACerts      = "vertica.com/mount-nma-certs"
-	MountNMACertsTrue  = "true"
-	MountNMACertsFalse = "false"
+	MountNMACertsAnnotation      = "vertica.com/mount-nma-certs"
+	MountNMACertsAnnotationTrue  = "true"
+	MountNMACertsAnnotationFalse = "false"
+
+	// This is a feature flag for running NMA process in a monolithic container together with
+	// vertica main process. When set to true the NMA process runs in a monolithic container
+	// together with vertica main process, when set to false it runs in a separate sidecar
+	// container.
+	RunNMAInMonolithicContainerAnnotation      = "vertica.com/run-nma-in-monolithic-container"
```

Comment:
I would suggest simply `run-nma-as-sidecar`, then the default would be `true`

Reply:
I agree with Roy. I like that annotation name better.

Reply:
done